raffler.php

```diff
@@ -4,6 +4,13 @@
 
 use PhpWvl\Command\RaffleCommand;
 use Symfony\Component\Console\Application;
+use Dotenv\Dotenv;
+
+try {
+    (new Dotenv(__DIR__))->load();
+} catch (Exception $exception) {
+    // meh
```

Comment:
If we are required to use the env information for twitter, then the exception is valid to throw here I guess

Reply:
It would be okay to let the user know something is wrong. But maybe instead of just throwing an exception, we should output a message with what's missing & where to fetch the twitter api credentials